src/components/elements/Inputs/FileInput/FilePreviewCard.tsx

```diff
@@ -70,36 +111,61 @@ const FilePreviewCard = ({ file, className, showPrivateCheckbox, onDelete, onPri
 interface UploadingProps {
   title: string;
   file: UploadedFile;
+  variant: FilePreviewCardVariant;
 }
 
-const Uploading = ({ title, file }: UploadingProps) => {
+const Uploading = ({ title, file, variant }: UploadingProps) => {
   const t = useT();
   const { format } = useFileSize();
+  const subtitleMap = {
+    image: "Image is being uploaded.",
+    geoFile: "Data is being uploaded."
+  };
 
+  let subtitle;
+  if (variant.type !== undefined && subtitleMap[variant.type] !== undefined) {
+    subtitle = subtitleMap[variant.type];
+  } else {
+    subtitle = "Uploading document ...";
+  }
   return (
     <FileCardContent
       title={title}
-      subtitle={`${format(file.size)} • ${t("Uploading document ...")}`}
+      variant={variant.fileCardContentVariant}
+      subtitle={`${format(file.size)} • ${t({ subtitle })}`}
```

Comment:
This use of `t` is a little weird; I'm surprised it works. Should simply be `t(subtitle)`

Reply:
done